pacttesting/testing.go

```diff
@@ -257,7 +256,7 @@ func AddPactInteraction(provider, consumer string, interaction *dsl.Interaction)
 	return pactServers[key].AddInteraction(interaction)
 }
 
-func VerifyInteractions(provider, consumer string, retryOptions ...retry.RetryOption) error {
+func VerifyInteractions(provider, consumer string, retryOptions ...retry.Option) error {
```

Comment:
Do we need to bump the major version for this change? It breaks the current contract by changing the type of retry option that the callers have to provide.

Reply:
I think we do, yes, should I do it in this PR?

Reply:
Yes, I've bumped major version